CMakeLists.txt

```diff
@@ -298,6 +297,7 @@ if(${SUBSURFACE_TARGET_EXECUTABLE} MATCHES "MobileExecutable")
 		mobile-widgets/qml/kirigami/src/libkirigami/platformtheme.cpp
 		subsurface-mobile-main.cpp
 		subsurface-mobile-helper.cpp
+		profile-widget/qmlprofile.cpp
```

Comment:
Interesting question.
Are we keeping all the qml / mobile UI related code in one place, or all profile code.
Not saying you are wrong, just pointing out the previous logic

Reply:
I follow your idea, but I think you forgot about map-widget/qml :-) that is not my work.

I have no problem having all qml code (and the related C++ code) in one place, I do however think it is easier to have the widgets (currently map and profile) as self sustained.